medusa/providers/torrent/html/scenetime.py

```diff
@@ -112,7 +113,7 @@ def parse(self, data, mode):
         items = []
 
         with BS4Parser(data, 'html5lib') as html:
-            torrent_rows = html.find_all('tr')
+            torrent_rows = html.find('div', {'id": 'torrenttable'}).find_all('tr')
```

Comment:
I think the preferred syntax is: `.find('div', id='torrenttable')`

Reply:
Also note the quotation marks don't match.

Reply:
Changed.